### ISSUE TITLE
Experimental deployments

### ISSUE THREAD

**Issue by Denubis** [googleartsculture/classifier-automl#1]
@justingrayston -- one of the routes we're taking to explore is to see if we can improve the training corpus to be more specific to time periods and sources. Is there a way we can apply for CloudML credits for academic research in this regard? Also, are there deployment instructions for the classifier?

**Comment by justingrayston** [comment]
We are writing up the docs on what we did with the classifier and CI/CD -  aiming to drop them into the repos in the next week or two, pending review and other commitments. 

I am not the Data Scientist here by any means - but one thing we have been looking at is a layer'd approach - it was discussed in different context, but it would probably work here.

You have a model that is trained purely to give a probability of the period, it doesn't understand or recognise anything other than that. If it has a period it is most confident about you then move onto the next model that has been trained on that period.

The context we were discussing was a possible next iteration where we remove most of the human work around the set up currently in the workbench.

If there is interest, we could look to arrange an online Meetup to discuss future roadmap, direction and chat through a few different topics. It is summer (allegedly) here and August is going to be tricky to squeeze in, but we could lock in early September?

**Comment by Denubis** [comment]
I agree that August is insane. Shall we take this to email? Our contact details should be on file. There are some other timing/people issues, but we can talk through them on email.
